logger.go

```diff
@@ -57,6 +59,17 @@ func NewWithWriter(serviceName string, w io.Writer, options ...Option) Logger {
 	zl = zl.Str("host", host).Str("release", os.Getenv("RELEASE"))
 	zl = zl.Str("service", serviceName).Str("name", serviceName)
 
+	// List of DataDog Metadata tags
+	var ddtags []string
+
+	// If we are in a container, populate ddtags with the containerId
+	if containerId, err := getContainerId(); err == nil {
+		ddtags = append(ddtags, fmt.Sprintf("container_id:%s", containerId))
+	}
+
+	// Add a zerolog field containing our datadog tags in the format "key1:value1,key2:value2,..."
+	zl = zl.Str("ddtags", strings.Join(ddtags[:], ","))
```

Comment:
if len of `ddtags` is 0 do we want to add this tag?
and `[:]` isn't needed:
```suggestion
	zl = zl.Str("ddtags", strings.Join(ddtags, ","))
```